### PR TITLE
Add document for che installation on eks

### PR DESCRIPTION
**Pull request by huonguyenlt**
## What does this pull request change?
[Installation guide on EKS #23209](https://github.com/eclipse-che/che/issues/23209)
## What issues does this pull request fix or reference?
main
## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+.Prerequisites
+* A Che domain name
+* A certificate for the che domain name: private and public keys in PEM format
+* An external OIDC provider, you need the following information from your provider: Issuer URL, Client ID, Client Secret
+
+.Install required components
+. Install link:https://kubernetes.github.io/ingress-nginx/[Ingress-Nginx Controller]
++
+[source,shell]
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc:18]: please check the Red Hat Style Guide section about commands with root privileges: https://redhat-documentation.github.io/supplementary-style-guide/#commands-with-root-privileges

you'll need to edit out mentions of shell in this doc

File: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc [hunk 2]
```diff
@@ -0,0 +1,105 @@
+:_content-type: PROCEDURE
+:navtitle: Installing {prod-short} on the {eks} cluster
+:description: Installing {prod-short} on the {eks} cluster
+:keywords: installing-{prod-short}-on-the-eks-cluster
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc:4]:
```suggestion
:keywords: installation, eks cluster
```

File: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+:page-aliases:
+
+[id="installing-{prod-short}-on-AWS-EKS.adoc"]
+= Installing {prod-short} on the {eks} cluster
+
+.Prerequisites
+* A Che domain name
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc:11]:
```suggestion
* A {prod-short} domain name
```

File: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+* A certificate for the che domain name: private and public keys in PEM format
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc:12]:
```suggestion
* A certificate (private and public keys in PEM format)  for the {prod-short} domain name
```

File: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+* An external OIDC provider, you need the following information from your provider: Issuer URL, Client ID, Client Secret
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc:13]:
```suggestion
* An external OpenId Connect (OIDC) provider
* An OIDC issuer URL
* An OICD client ID
* An OICD client secret
```

File: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+.Install required components
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc:15]:
```suggestion
.Installing required components
```

File: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+----
+
+. Install the link:https://github.com/eclipse-che/che-operator[eclipse-che-operator]
++
+[source,shell]
+----
+kubectl apply -f https://raw.githubusercontent.com/eclipse-che/che-operator/refs/heads/main/deploy/deployment/kubernetes/combined.yaml
+----
+
+.Integrate the external OIDC provider with the EKS cluster
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc:38]:
```suggestion
.Integrating the external OIDC provider with the EKS cluster
```

File: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+* In the OIDC Identity Providers section, select Associate Identity Provider.
+* On the Associate OIDC Identity Provider page, enter or select the following options, and then select Associate.
+[subs="+quotes,attributes"]
+----
+For Name, enter a unique name for the provider.
+For Issuer URL, enter the URL for your provider
+For Client ID, enter the OIDC identity provider's client ID 
+For Username claim, enter `email`
+----
+.Create secret for the seft sign certificate
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc:49]:
```suggestion
.Creating secret for the seft-sign certificate
```

File: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
++
+[source,shell]
+----
+kubectl apply -f https://raw.githubusercontent.com/eclipse-che/che-operator/refs/heads/main/deploy/deployment/kubernetes/combined.yaml
+----
+
+.Integrate the external OIDC provider with the EKS cluster
+* Select your EKS cluster, and then select the `Access` tab.
+* In the OIDC Identity Providers section, select Associate Identity Provider.
+* On the Associate OIDC Identity Provider page, enter or select the following options, and then select Associate.
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc:41]: what's the correct name of the page, please?  Associate OIDC Identity Provider or  Associate Identity Provider

File: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+----
+
+. Add the required labels to the secret:
++
+[subs="+quotes,attributes"]
+----
+$ {orch-cli} label secret che-tls app.kubernetes.io/part-of=che.eclipse.org -n {prod-namespace}
+----
+
+.Deploy che cluster
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc:67]:
```suggestion
.Deploying {prod-short} cluster
```

File: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+. Prepare `CheCluster` manifest file
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc:68]:
```suggestion
. Prepare the `CheCluster` manifest file
```

File: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc [hunk 12]
```diff
@@ -0,0 +1,105 @@
+:_content-type: PROCEDURE
+:navtitle: Installing {prod-short} on the {eks} cluster
+:description: Installing {prod-short} on the {eks} cluster
+:keywords: installing-{prod-short}-on-the-eks-cluster
+:page-aliases:
+
+[id="installing-{prod-short}-on-AWS-EKS.adoc"]
+= Installing {prod-short} on the {eks} cluster
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc:8]:
```suggestion
= Installing {prod-short} on an EKS cluster
```

File: modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+
```
Comment (deerskindoll) [inline modules/administration-guide/pages/installing-che-on-AWS-EKS.adoc:9]: docs usually have a short introductory paragraph about the process they describe. can you add it, please?

examples: 

- https://eclipse.dev/che/docs/stable/administration-guide/installing-che-on-the-virtual-kubernetes-cluster/
- https://eclipse.dev/che/docs/stable/administration-guide/installing-che-on-microsoft-azure/
- https://eclipse.dev/che/docs/stable/administration-guide/installing-che-on-openshift-using-cli/

**Review by deerskindoll — COMMENTED** [review]
first batch of suggestions. please check them out, edit the doc as necessary and I'll have a second look at it.

**Comment by ibuziuk** [comment]
@tolusha please, review as part of the https://github.com/eclipse-che/che-docs/pull/2808

**Review by ibuziuk — COMMENTED** [review]
@huonguyenlt thank you for the contribution
Please, submit the ECA form required by the Eclipse Foundation - https://www.eclipse.org/legal/eca/

**Comment by tolusha** [comment]
@huonguyenlt 
Let me create another PR describing installing Che on EKS based on yours.

**Comment by huonguyenlt** [comment]
@tolusha does this means that I dont have to update this pr as the above review comments any more?

**Comment by tolusha** [comment]
Yes, I will do the rest if you don't mind

**Comment by huonguyenlt** [comment]
It's ok to me

**Comment by tolusha** [comment]
Closed in favor of https://github.com/eclipse-che/che-docs/pull/2828

**Pull request closed without merging**